pkg/authenticator/kubeconfig_test.go

```diff
@@ -35,6 +35,7 @@ func TestManagerContext_getRegistry(t *testing.T) {
 		fileName, err := sut.GetKubeconfig(ctx, "billy")
 		assert.NoError(t, err)
 		assert.Equal(t, "billy-kubeconfig", fileName)
-		os.Remove(fileName)
+		err = os.Remove(fileName)
+		assert.NoError(t, err)
```

Comment:
Should we use `require.NoError` more consistently across different tests?

Reply:
This is just informative on the clean-up. The require.NoError stops the tests which would be fine here, but this error is not very important